### PR TITLE
Make manimpango work with MarkupText

### PR DESCRIPTION
**Pull request by PhilippImhof**
Added `MarkupUtils` class (for separation) with static `text2svg` method in order to be able to use `manimpango` with the new `MarkupText` class -- as discussed on Discord

### CODE REVIEW

File: manimpango/cmanimpango.pyx [hunk 1]
```diff
@@ -256,3 +256,88 @@ def text2svg(
...
+            cairo_surface_destroy(surface)
+            raise MemoryError("Cairo.Context can't be created.")
+        elif status != CAIRO_STATUS_SUCCESS:
+            cairo_destroy(context)
+            cairo_surface_destroy(surface)
+            raise Exception(cairo_status_to_string(status))
+
+        cairo_move_to(context,START_X,START_Y)
+        cdef PangoLayout* layout
+        cdef double width_layout = width
```
Comment (naveen521kk) [inline manimpango/cmanimpango.pyx:300]: Usually, people define the variable type at the top of the class or function.

File: manimpango/cmanimpango.pyx [hunk 2]
```diff
@@ -256,3 +256,88 @@ def text2svg(
...
+        slant: str,
+        weight: str,
+        size:int,
+        line_spacing:int,
+        disable_liga:bool,
+        file_name:str,
+        START_X:int,
+        START_Y:int,
+        width:int,
+        height:int,
```
Comment (naveen521kk) [inline manimpango/cmanimpango.pyx:273]:
```suggestion
        text: str,
        font: str,
        slant: str,
        weight: str,
        size: int,
        line_spacing: int,
        disable_liga: bool,
        file_name: str,
        START_X: int,
        START_Y: int,
        width: int,
        height: int,
```

**Review by naveen521kk — CHANGES_REQUESTED** [review]

File: manimpango/cmanimpango.pyx (hunk 2 above)
Comment (PhilippImhof) [reply manimpango/cmanimpango.pyx]: Thanks. My editor refused to run `black` and I did not think to do it manually.

Comment (naveen521kk) [reply manimpango/cmanimpango.pyx]: Black doesn't work for `.pyx` files sadly.

Comment (PhilippImhof) [reply manimpango/cmanimpango.pyx]: I found that out just now, while I was looking for a way to change the editor's config :)

File: manimpango/cmanimpango.pyx [hunk 3]
```diff
@@ -256,3 +256,90 @@ def text2svg(
...
+    ) -> int:
+        file_name_bytes = file_name.encode("utf-8")
+
+        cdef cairo_surface_t* surface
+        cdef cairo_t* context
+        cdef PangoFontDescription* font_desc
+        cdef PangoLayout* layout
+        cdef cairo_status_t status
+        cdef double width_layout = width
+        cdef double font_size = size
```
Comment (naveen521kk) [inline manimpango/cmanimpango.pyx:283]:
```suggestion
        cdef cairo_surface_t* surface
        cdef cairo_t* context
        cdef PangoFontDescription* font_desc
        cdef PangoLayout* layout
        cdef cairo_status_t status
        cdef double width_layout = width
        cdef double font_size = size
        
        file_name_bytes = file_name.encode("utf-8")
```

**Comment by naveen521kk** [comment]
Let checks pass and I will merge and release it.

**Review by naveen521kk — APPROVED** [review]
LGTM!

**Pull request merged**